### PR TITLE
:sparkles: Adding Benefit Section

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206109714345729

### CODE REVIEW

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { BenefitCardProps } from './benefit-card.types';
+
+const BenefitCard: FC<BenefitCardProps> = ({ title, description, Icon }) => (
+  <Box
```
Comment (MarioBatalha) [inline views/institutional/components/benefits/benefit-card/index.tsx]: Is missing border

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    p="2.5rem"
+    display="flex"
+    height="27rem"
+    borderRadius="2rem"
+    flexDirection="column"
+    border="2px solid #FFF"
```
Comment (KipandaJr) [inline views/institutional/components/benefits/benefit-card/index.tsx]: border here @MarioBatalha

Comment (MarioBatalha) [reply views/institutional/components/benefits/benefit-card/index.tsx]: Yes, take a look on figma. I'm not sure if is border or box-shadow.

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { BenefitCardProps } from './benefit-card.types';
+
+const BenefitCard: FC<BenefitCardProps> = ({ title, description, Icon }) => (
+  <Box
+    p="2.5rem"
```
Comment (atilasousa) [inline views/institutional/components/benefits/benefit-card/index.tsx]: check the ui kit

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    borderRadius="2rem"
+    flexDirection="column"
+    border="2px solid #FFF"
+    justifyContent="space-between"
+    background="linear-gradient(36deg, #FFF 2.01%, rgba(255, 255, 255, 0.16) 98.41%)"
+  >
+    <Typography
+      variant="title"
+      size="medium"
+      fontSize="2rem"
```
Comment (atilasousa) [inline views/institutional/components/benefits/benefit-card/index.tsx]: why font size??

i think the component already comes with default sizes

**Review by atilasousa — COMMENTED** [review]
borders are missing, they're not like in the figma

Screenshot 2023-12-13 at 23 02 10

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <Box
+      p="2.5rem"
+      display="flex"
+      minHeight="27rem"
+      borderRadius="2rem"
+      flexDirection="column"
+      justifyContent="space-between"
+      border={`2px solid ${theme.colors.onPrimary}`}
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/benefit-card/index.tsx]: use `borderColor` instead

File: views/institutional/components/benefits/group-card.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+import BenefitsCard from './benefit-card';
+import CoinsIllustration from './illustration/coins-illustration';
+import { DAOSVG, FarmingSVG } from './illustration/illustration-svg';
+
+const BenefitGroupCard: FC = () => (
+  <Box
+    flexDirection="column"
+    display={['flex', 'flex', 'flex', 'grid']}
+    gap="1.5rem"
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/group-card.tsx]: replace for Theme gap keys

File: views/institutional/components/benefits/group-card.tsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    gridTemplateColumns="repeat(3, 1fr)"
+  >
+    <BenefitsCard
+      title="Airdrop"
+      Icon={
+        <Box width="100%" position="relative" height="20rem" my="1rem">
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/group-card.tsx]: `my` has theme key

File: views/institutional/components/benefits/header.tsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
+import { Box, Tag, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+const BenefitHeader: FC = () => (
+  <Box
+    mt="2xl"
+    py={['2rem', '2rem', '2rem', '4rem']}
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/header.tsx]: replace for theme space `keys`

File: views/institutional/components/benefits/header.tsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    mx="auto"
+    width={['100%', '100%', '100%', '40rem']}
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/header.tsx]: becareful with fixed width

File: views/institutional/components/benefits/header.tsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    display="flex"
+    alignItems="center"
+    flexDirection="column"
+    justifyContent="center"
+  >
+    <Tag variant="outline" size="medium" width="max-content" mb="2rem">
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/header.tsx]: `mb` has theme space key

File: views/institutional/components/benefits/header.tsx [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+      Unique Feature
+    </Tag>
+    <Typography
+      size="large"
+      mb="1rem"
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/header.tsx]: `mb` has theme space key

File: views/institutional/components/benefits/header.tsx [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+      variant="title"
+      fontWeight="700"
+      fontStyle="normal"
+      textAlign="center"
+      fontSize={['2.25rem', '2.25rem', '2.25rem', '2.8125rem']}
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/header.tsx]: `fontSize` and `lineHeight` have theme keys

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+      background={`linear-gradient(36deg, ${theme.colors.onPrimary} 2.01%, rgba(255, 255, 255, 0.16) 98.41%)`}
+    >
+      <Typography
+        variant="title"
+        size="medium"
+        fontSize="2rem"
```
Comment (MarioBatalha) [inline views/institutional/components/benefits/benefit-card/index.tsx]: replaces all this sizes for theme keys (fontSize, fontWeight and lineHeight)

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/institutional/components/benefits/benefit-card/index.tsx [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+const BenefitCard: FC<BenefitCardProps> = ({ title, description, Icon }) => {
+  const theme = useTheme() as Theme;
+
+  return (
+    <Box
+      p="3xl"
+      display="flex"
+      minHeight="27rem"
+      border="2px solid"
+      borderRadius="2rem"
```
Comment (git-marcopitra) [inline views/institutional/components/benefits/benefit-card/index.tsx:15]: borderRadius has a key theme

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**